src/main/scala/uk/co/telegraph/traveltours/clients/SnsClient.scala

```
@@ -4,58 +4,40 @@ import javax.inject.Inject
 
 import akka.actor.ActorSystem
 import com.amazonaws.services.sns.AmazonSNSClientBuilder
-import org.joda.time.DateTime
-import play.api.Configuration
-import play.api.libs.json.Json
+import org.json4s.DefaultFormats
+import org.json4s.native.Serialization.write
 import uk.co.telegraph.traveltours.models.{SnsMessage, Tour}
 import uk.co.telegraph.utils.client.GenericClient
-import uk.co.telegraph.utils.client.models.{ClientDetails, ClientFailureDetails}
+import uk.co.telegraph.utils.client.models.ClientDetails
 
-import scala.concurrent.Future
+import scala.concurrent._
 import scala.concurrent.duration.FiniteDuration
+import scala.util.{Failure, Success, Try}
 
 object SnsClient {
   val client = AmazonSNSClientBuilder.defaultClient()
 }
 
-class SnsClient @Inject() (config:Configuration)(implicit val system: ActorSystem) extends GenericClient {
+class SnsClient @Inject()()(implicit val system: ActorSystem) extends GenericClient {
 
   import system.dispatcher
+  implicit val formats = DefaultFormats
 
-  lazy val topicArn = config.getString("app.sns.topicArn").getOrElse("")
+  lazy val clientConfig = system.settings.config.getConfig("app.sns")
+  lazy val topicArn = clientConfig.getString("topicArn")
 
   @throws(classOf[Exception])
   def publish(tour: Tour): Unit = {
     val message = SnsMessage(tour.flakeId.get, "Tour", System.currentTimeMillis(), "", tour.status)
-    SnsClient.client.publish(topicArn, Json.toJson(message).toString)
+    SnsClient.client.publish(topicArn, write(message))
```

I'll bubble exceptions up instead of returning None if an operation fails